### PR TITLE
Release highlights

### PR DESCRIPTION
**Pull request by RealHarshThakur**
Signed-off-by: Harsh Thakur <harshthakur9030@gmail.com>

### CODE REVIEW

File: docs/content/en/docs/releases/1.13.0.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ ImagePullPolicy and PodSpec are parameters which can be used to control the Kube
 #### RequestsPerPod and OnceOnly fields are added for pool manager
 
 RequestsPerPod will denote how many requests will be routed to each pod in Pool Manager.
-OnceOnly is similar to a Kubernetes Job, it is useful in long-running tasks where only one request is served by one pod.
+OnceOnly is similar to a Kubernetes Job, it is useful in long-running tasks where only one request is served by one pod. You can read more about it [here](https://docs.fission.io/docs/usage/executor/)
 
 #### Websocket support for reaping
 
-With recent changes in the NodeJS environment for websocket, conventional timeout based reaping wasn't applicable so an event-based reaping mechanism to support Websocket was introduced.
+With recent changes in the NodeJS environment for websocket, conventional timeout based reaping wasn't applicable so an event-based reaping mechanism to support Websocket was introduced. You can try out the example [here](https://github.com/fission/examples/tree/master/samples/websocket)
```
Comment (RealHarshThakur) [inline docs/content/en/docs/releases/1.13.0.md:28]: TODO:
Update the README in examples repo to the blog when it's merged:
https://github.com/fission/blog.fission.io/pull/69

Comment (sanketsudake) [reply docs/content/en/docs/releases/1.13.0.md:28]: cc @gauravgahlot

File: docs/content/en/docs/releases/1.13.0.md [hunk 2]
```diff
@@ -8,24 +8,24 @@ weight: 90
...
+Fission support only one HTTP verb per request route. Now we enable multiple HTTP verbs per route. The same function now can serve GET, POST or different types of requests. Here's an example you can try out: [Nextjs example]( https://github.com/fission/examples/tree/master/samples/nextjs-prefixpath )
 
 #### Added ImagePullPolicy and PodSpec for KEDA connector deployments
 
-ImagePullPolicy and PodSpec are parameters which can be used to control the Kubernetes deployment of pods which used by KEDA connector. For example, with PodSpec, taints and toleration would be possible.
+ImagePullPolicy and PodSpec are parameters which can be used to control the Kubernetes deployment of pods which used by KEDA connector. For example, with PodSpec, taints and toleration would be possible, similar to [this](https://docs.fission.io/docs/spec/podspec/toleration/) can be done. You can learn more about Pod Spec [here](https://docs.fission.io/docs/spec/). If you want to try PodSpec in MQT, you can check this example [out](https://github.com/fission/examples/tree/master/samples/kafka-keda/specs).
```
Comment (sanketsudake) [inline docs/content/en/docs/releases/1.13.0.md:19]: `you can checkout [this example]`

File: docs/content/en/docs/releases/1.13.0.md [hunk 3]
```diff
@@ -8,24 +8,24 @@ weight: 90
 
 #### Support for prefix-based routes for functions
 
-We can forward all request matching a particular prefix to the function which supports routing internally. This enables different functions which can handle multiple requests with prefix route.
+We can forward all request matching a particular prefix to the function which supports routing internally. This enables different functions which can handle multiple requests with prefix route. Here's an example you can try out: [Nextjs example]( https://github.com/fission/examples/tree/master/samples/nextjs-prefixpath )
```
Comment (sanketsudake) [inline docs/content/en/docs/releases/1.13.0.md:11]: `You can refer [NextJs example]` 

Same below.

**Review by sanketsudake — APPROVED** [review]

**Pull request merged**